scripts.Rmd

```diff
@@ -0,0 +1,318 @@
+# `drake` and script-based workflows {#scripts}
+
+```{r, message = FALSE, warning = FALSE, echo = FALSE}
+knitr::opts_knit$set(root.dir = fs::dir_create(tempfile()))
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+```{r, message = FALSE, warning = FALSE, echo = FALSE}
+library(drake)
+library(glue)
+library(purrr)
+library(rlang)
+library(tidyverse)
+invisible(drake_example("script-based-workflows", overwrite = TRUE))
+tmp <- file.copy(
+  list.files("script-based-workflows/R/",
+             pattern = "*.R", full.names = TRUE),
+  ".",
+  recursive = TRUE)
+tmp <- file.copy("script-based-workflows/raw_data.xlsx", ".")
+tmp <- file.copy("script-based-workflows/report.Rmd", ".")
+dir.create("data")
+```
+## Function-oriented workflows
+
+`drake` was built around the idea that writing functions for data analysis allows
```

Comment:
This chapter is really coming along nicely, so I am going to start being pickier about rhetoric.

This sentence seems a bit long and cumbersome for an intro. How about:

> `drake` works best when you write functions for data analysis. Functions break down complicated ideas into manageable pieces.